### DISCUSSION TITLE
Searching within the filter web part does not show additional items

### DISCUSSION THREAD

**Discussion by Linden-Daniels** [microsoft-search/pnp-modern-search#2923] [Q&A]
**Version used**
v4.8

**Describe the bug**
When performing a search on a web page that has the search filters web part, we have defined several filters. We can see the search results and search filters are populated. We have limited each search filters to 5 items. For one filter (e.g. Matter Name), we know that many documents have been tagged with "SMH XXX". 

The Filter web part only shows the first 5 items for Matter Name and "SMH XXX" is not in the initial list of 5. In the Filter web part, under the Matter Name Filter, we try to search for "SMH XXX" but it says no results found.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Create a Column e.g. Matter Name and tag many documents with various values (Use more than 10 values for Matter Name)
2. Configure a search page with a search results web part and search filter web part
3. Add Matter Name to the Filters, limit the number items to display to 5
4. Do a search. The results must include documents with more than 5 distinct Matter Names
5. The Search Filter web part, must show the Matter Name filter and there must be 5 Matter Names shown 
6. In the Search Filter web part, click in the search box under Matter Name
7. Type in a Matter Name that is not listed below, it will state "No results found"

**Expected behavior**
When you type in a value in a filter, it should return matching values that are not immediately displayed (In the top 5 results)

**Screenshots**
![searchfilternotfound](https://user-images.githubusercontent.com/56667476/233533499-e86b37e0-8517-4284-bb71-05b4a9168cb2.png)
![6dd6a072-d9ab-4ec1-a10e-f8ec92e1a677](https://user-images.githubusercontent.com/56667476/233533820-22a5582c-6cbb-44ac-832a-b1446ff35fe2.jpg)

**Desktop (please complete the following information):**
 Chrome 112.0.5615.121 
We have been able to replicate this issue on 2 tenants (One has v4.7 and the other has v4.8)

**Comment by wobba** [comment]
This is expected. The number 5 you input is how many values to get back on the API call. The on-page filtering just filters on those items. So, you should increase 5 to a number covering your expected maximum.

**Reply by Linden-Daniels** [reply]
Brainstormed this with the customer and the workaround we came up with was to use a ComboBox rather than a CheckBox. We then set the # of values to 200 items. This way the list of filters is not too long and we can search a large set of filters.
The only downside is the user has to click on the combobox to see the filters - So one more click for them.

**Reply by wobba** [reply]
The other alternative is to fund development time to add what you need ;)
